.gitignore

```diff
@@ -139,3 +139,5 @@ hydrolib/dhydamo/debug.py
 .DS_Store
 coverage-reports
 xunit-reports
+hydrolib/tests/model/*
+dimr_config.xml
```

Comment:
For hydrolib-core, we have in `.gitignore`:
```
# Testdata
tests/data/output
```

Reply:
This is indeed the location where the resulting model is written. dimr_config is separate because that's by default written twice somewhere else (see issue #59)